Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/TestPages/TestPages.cs

```diff
@@ -169,7 +170,9 @@ public static void NavigateToIssue(Type type, IApp app)
 
 			int maxAttempts = 2;
 			int attempts = 0;
-
+#if __WINDOWS__
+			bool attemptOneRestart = false;
```

Comment:
Thank you!

Reply:
Your welcome, I owe you guys for the number of times I've forgotten #if App in an issue page!